src/libcadet/model/exchange/LinearExchange.cpp

```diff
@@ -0,0 +1,267 @@
+// =============================================================================
+//  CADET
+//  
+//  Copyright © 2008-2024: The CADET Authors
+//            Please see the AUTHORS and CONTRIBUTORS file.
+//  
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the GNU Public License v3.0 (or, at
+//  your option, any later version) which accompanies this distribution, and
+//  is available at http://www.gnu.org/licenses/gpl.html
+// =============================================================================
+
+
+
+#include "model/BindingModel.hpp"
+#include "model/ExternalFunctionSupport.hpp"
+#include "ParamIdUtil.hpp"
+#include "model/ModelUtils.hpp"
+#include "cadet/Exceptions.hpp"
+#include "model/Parameters.hpp"
+#include "LocalVector.hpp"
+#include "SimulationTypes.hpp"
+#include "Memory.hpp"
+#include "model/PhaseTransitionModel.hpp"
+#include "ParamReaderHelper.hpp"
+#include "model/parts/MultiChannelConvectionDispersionOperator.hpp"
+
+#include <vector>
+#include <unordered_map>
+#include <functional>
+#include <algorithm>
+#include <iterator>
+#include <tuple>
+
+namespace cadet
+{
+
+namespace model
+{
+/**
+ * @brief Defines the linear exchange model
+ * @details Implements the linear exchange model for the MCT model
+ * The exchange is given by a matrix of exchange coefficients eij for each component i and j and the chross Ai section from channel i.
```

Comment:
please use actual math notation like e_{ij} instead of eij
Also, it should be "cross-section A_i", not "chross Ai section"